packages/dashboard/src/components/querySettingsSync/utils.ts

```diff
@@ -0,0 +1,14 @@
+export const selectIntialRefreshRate = (refreshRate?: number) => {
+  if (refreshRate === 1000) {
+    return { label: '1s', value: '1000' };
+  } else if (refreshRate === 5000) {
+    return { label: '5s', value: '5000' };
+  } else if (refreshRate === 10000) {
+    return { label: '10s', value: '10000' };
+  } else if (refreshRate === 60000) {
+    return { label: '1m', value: '60000' };
+  } else if (refreshRate === 300000) {
+    return { label: '5m', value: '300000' };
+  }
+  return { label: '5s', value: '5000' };
+};
```

Comment:
Try using a map object instead of a bunch of conditionals to clean this code up.